app/assets/stylesheets/hackweek.css.scss

```diff
@@ -10,8 +10,24 @@ body {
   background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1IiBoZWlnaHQ9IjUiPgo8cmVjdCB3aWR0aD0iNSIgaGVpZ2h0PSI1IiBmaWxsPSIjZmZmIj48L3JlY3Q+CjxyZWN0IHdpZHRoPSIxIiBoZWlnaHQ9IjEiIGZpbGw9IiNjY2MiPjwvcmVjdD4KPC9zdmc+");
 }
 
+h4.project-list-item {
+  font-size: 24px;
+  font-weight: normal;
+}
+
+h4.project-list-item img {
+  width: 26px;
+  margin-right: 10px;
+  max-height: 30px;
+}
+
+p {
+    margin: 15px 0 0 0;
+    font-size: 16px;
```

Comment:
Please do not just change HTML tags layout. You'll never know where `<p>` will be used